keras_cv/visualization/plot_image_gallery.py

```diff
@@ -179,16 +179,18 @@ def plot_image_gallery(
             current_axis.axis("off")
 
     if path is None and not show:
```

Comment:
I think this behavior was correct before -- we only return the plot if the user isn't showing or saving it